cfme/networks/views.py

```diff
@@ -783,6 +784,7 @@ def is_displayed(self):
 
 class FloatingIpDetailsView(BaseLoggedInPage):
     """ Represents detail view of floating ip """
+    title = Text('//div[@id="main-content"]//h1')
```

Comment:
I think the title should be within entities, so these view structure are consistent.